c2ctask/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { ContainerMapping } from './containerMapping';
+import { Inputs } from "./common";
+
+async function run() {
+    const inputString: string = tl.getInputRequired(Inputs.TaskType);
+    console.log('Running ', inputString);
+    const conMap = new ContainerMapping(inputString);
+    conMap.run();
+}
+
+run().catch(error => {
+    console.log("Ran into error: " + error);
+    // Always mark it as success even on error
+    tl.setResult(tl.TaskResult.Succeeded, "Finished execution", true);

```

Comment:
Can we not set the description of the error based on the outcome.
Finished execution successfully
Finished execution with error ?